### PR TITLE
Print warning for deprecated testnets

### LINKED ISSUE #4163: Add deprecation warning for Ropsten, Kiln, Rinkeby
The 3 testnets will be deprecated in the near future. I propose to print a deprecation warning in the logs at startup to inform the user.

The deprecation dates can be found here: https://blog.ethereum.org/2022/06/21/testnet-deprecation/

### PR DESCRIPTION
**Pull request by daniellehrner**
Signed-off-by: Daniel Lehrner <daniel.lehrner@consensys.net>

## PR description

Prints a deprecation warning for Ropsten, Kiln and Rinkeby

## Fixed Issue(s)
fixes #4163

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if
    [updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).

The documentation should mention that Ropsten, Rinkeby and Kiln are deprecated

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: besu/src/test/java/org/hyperledger/besu/cli/NetworkDeprecationMessageTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      names = {"RINKEBY", "ROPSTEN", "KILN"})
+  void shouldGenerateDeprecationMessageForDeprecatedNetworks(final NetworkName network) {
+    assertThat(NetworkDeprecationMessage.generate(network))
+        .contains(network.humanReadableNetworkName() + " is deprecated and will be shutdown");
+  }
+
+  @ParameterizedTest
+  @EnumSource(
+      value = NetworkName.class,
+      names = {"RINKEBY", "ROPSTEN", "KILN"},
```
Comment (jflo) [inline besu/src/test/java/org/hyperledger/besu/cli/NetworkDeprecationMessageTest.java:39]: Shouldn't this be MAINNET, SEPOLIA, GOERLI ?

**Review by jflo — CHANGES_REQUESTED** [review]
The 2nd test on throwing error for non-deprecated networks is only run against deprecated networks. I think the input enums should be switched to MAINNET, SEPOLIA, GOERLI

File: besu/src/test/java/org/hyperledger/besu/cli/NetworkDeprecationMessageTest.java (hunk 1 above)
Comment (daniellehrner) [reply besu/src/test/java/org/hyperledger/besu/cli/NetworkDeprecationMessageTest.java]: I've made it more readble

**Review by jflo — APPROVED** [review]
thx!

**Pull request merged**